cypress/e2e/touringSeriesDetailPage.cy.js

```diff
@@ -0,0 +1,7 @@
+describe('Touring Series Detail page', () => {
+  it('Visit the Touring Series detail page', () => {
+    cy.visit('/touring-series/through-indian-eyes-native-american-cinema')
+
+    cy.percySnapshot('touringseriesdetailpage', { widths: [768, 992, 1200] })
```

Comment:
@jendiamond  remove the widths for PercySnapshot, we decided not to use widths in the tests.